.vscode/launch.json

```diff
@@ -48,22 +48,42 @@
     },
     {
       "type": "node",
-      "name": "Zowe Explorer FTP Unit Tests (Jest)",
+      "name": "FTP Unit Tests (Jest)",
       "request": "launch",
       "runtimeArgs": ["--inspect-brk", "${workspaceFolder}/node_modules/jest/bin/jest", "-i"],
       "cwd": "${workspaceFolder}/packages/zowe-explorer-ftp-extension",
       "console": "integratedTerminal",
       "sourceMaps": true
     },
     {
-      "name": "Zowe Explorer Integration Tests (Mocha)",
+      "type": "node",
+      "name": "End-to-end Tests",
+      "request": "launch",
+      "runtimeExecutable": "pnpm",
```

Comment:
Without having to add a completely new launch-configuration... could we support headless environments? 😓 
If we think it's too much work, I'm happy to create my own set of launch configs outside of the project 😋

Reply:
That's a good idea, I'll look into it 🙂

Reply:
maybe another time 😋